### ISSUE TITLE
How to set TilteFont for Undercoated Dialogs

### ISSUE THREAD

**Issue by iamchathu** [mgarin/weblaf#301]
It seems Title Font of UndecoratedDialogs is not support Sinhala Unicode.How to Change That font globally.

**Comment by mgarin** [comment]
Custom decoration title doesn't have any option to setup font yet, it will have one with later update for better fonts customization.

**Comment by mgarin** [comment]
Starting with the release of v1.29 it will be possible to set it in `WebLookAndFeel.globalTitleFont`, with later updates that will be replaced with more convenient ways to setup and manage application fonts.

**Comment by iamchathu** [comment]
Can I set Font using

```java
WebLookAndFeel.globalTitleFont = New Font(...)
```
in 1.29 pre-release ?

**Comment by iamchathu** [comment]
I tried it with 1.29 it is not working :cry:

**Comment by mgarin** [comment]
That is why I said "with release" - it is not yet available in pre-release version and will only be available in the final release after other major changes are included.

**Comment by iamchathu** [comment]
Opps. I try to Find workaround.

**Comment by iamchathu** [comment]
After i change 1.28 to 1.29 pre release 

```java
spnTblInvoice.getViewport().setBackground(new Color(57, 66, 100));

```

is not working the background of table is set to white

**Comment by mgarin** [comment]
Will be closing this one, it can be tested on snapshot builds of the current `styling` branch.
It will also be included in the final v1.2.9 release.
